lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -69,6 +69,19 @@ def self.deliver_queue(args, options = {})
 
   def self.deliver(*args)
     options     = options_from_args(args)
+
+    # HACK: If we are launching a specific Automate instance representing a workflow, then we run that instead
+    if options[:namespace] == "System" && options[:class_name] == "Workflow"
+      workflow_id = options[:instance_name].split("-").last.to_i
+      workflow = Workflow.find(workflow_id)
+
+      task_id = workflow.execute(inputs: options)
```

Comment:
Huh. never thought I'd see the day that Jason would use JSON syntax over hash rockets

Reply:
I've always said that this format makes sense for _keyword parameters_ which is what the syntax was designed for.  I'm against it for regular Hashes. IMO, Ruby should have made a distinction between Hashes and kwargs because they are different things. 😁